addons/website/static/src/snippets/s_mega_menu_little_icons/000.scss

```diff
@@ -6,21 +6,7 @@
         }
     }
 
-    .s_mega_menu_gray_area:last-child {
```

Comment:
You lost the `:last-child` part, thus changing the stable style, we can't :wink: Same at the other location